app/src/main/java/de/tum/in/tumcampusapp/component/ui/chat/di/ChatComponent.kt

```diff
@@ -0,0 +1,29 @@
+package de.tum.`in`.tumcampusapp.component.ui.chat.di
+
+import dagger.Subcomponent
+import de.tum.`in`.tumcampusapp.component.ui.chat.activity.ChatActivity
+import de.tum.`in`.tumcampusapp.component.ui.chat.activity.ChatRoomsActivity
+import de.tum.`in`.tumcampusapp.component.ui.onboarding.WizNavExtrasActivity
+import de.tum.`in`.tumcampusapp.service.SendMessageService
+
+@Subcomponent(modules = [ChatModule::class])
+interface ChatComponent {
```

Comment:
document

Reply:
What do you imagine to be the documentation here?

Reply:
Why did you create it?
What's its purpose?